public/index.js

```diff
@@ -308,6 +309,19 @@
     http.setRequestHeader("Content-type", "application/json");
     var payload = JSON.stringify({emailAddress:emailAddress,
       sessionStorage:sessionStorage});
+
+    http.onreadystatechange = function() {
+      if (http.readyState === 4) {
+        var response = JSON.parse(http.responseText);
+          if (response.status === 'Email sent'){
+            finishPrompt.innerHTML = '<div class="checkmark"></div>' + response.status;
```

Comment:
I do not think usage of innerHTML is advisable.

use `document.createElement` and friends .

Ie https://developer.mozilla.org/en-US/docs/Web/API/Document_object_model/How_to_create_a_DOM_tree

Also, break out the logic of making an ajax request into its own function.